pkg/utils/k8sutil_test.go

```diff
@@ -248,6 +256,59 @@ func TestEnsureService(t *testing.T) {
 	}
 }
 
+func initializeConfigmap(clientset *fake.Clientset, lr *langruntime.Langruntimes) {
```

Comment:
Moving the code to a common package (eg. testUtils) is tricky because it would inject cyclic dependency. As the `testUtils` package would depend on `langruntime` package and vice-versa.

Oneway to solve this is to create a file called `testhelper.go` inside langruntime package and put the code here because golang does not export functions in `_test` file